project.clj

```diff
@@ -125,7 +125,7 @@
                          [puppetlabs/rbac-client ~rbac-client-version :classifier "test"]
                          [puppetlabs/analytics-client "1.0.2"]
                          [puppetlabs/clj-shell-utils "1.0.2"]
-                         [puppetlabs/jruby-utils "2.2.0"]
+                         [puppetlabs/jruby-utils "2.3.0"]
```

Comment:
Shouldn't this be 2.3.1? It looks like 2.3.0 is one that I updated (but forgot to actually release) that includes JRuby 9.2.9? https://github.com/puppetlabs/jruby-utils/commits/master